incubator/cassandra/values.yaml

```diff
@@ -171,8 +171,8 @@ backup:
     iam.amazonaws.com/role: cain
 
   image:
-    repos: nuvo/cain
-    tag: 0.4.1
+    repository: maorfr/cain
+    tag: 0.6.0
 
   # Additional arguments for cain
   # Ref: https://github.com/nuvo/cain#usage
```

Comment:
this has to turn to https://github.com/maorfr/cain#usage

and any other places such as this one.